playwright-e2e/pages/other-proceedings.ts

```diff
@@ -0,0 +1,24 @@
+import { type Page, type Locator, expect } from "@playwright/test";
+
+export class OtherProceedings {
+  readonly page: Page;
+  readonly otherProceedingsHeading: Locator;
+  readonly areThereAnyPastOrOngoingProccedingsReleventToCase: Locator;
+  readonly continueButton: Locator;
+  readonly saveAndContinueButton: Locator;
+
+  public constructor(page: Page) {
+    this.page = page;
+    this.otherProceedingsHeading = page.getByRole('heading', { name: 'Other Proceedings' });
+    this.areThereAnyPastOrOngoingProccedingsReleventToCase = page.getByRole('radio',  { name: 'No', exact: true });
+    this.continueButton = page.getByRole('button', { name: 'Continue' });
```

Comment:
continueButton and saveAndContinue buttons are defined in base page, may be re-use it rather than defining it on each page?

Reply:
True